Source/Flow/Public/FlowComponent.h

```diff
@@ -203,9 +203,23 @@ class FLOW_API UFlowComponent : public UActorComponent
 	UFUNCTION(BlueprintPure, Category = "RootFlow", meta = (DeprecatedFunction, DeprecationMessage="Use GetRootInstances() instead."))
 	UFlowAsset* GetRootFlowInstance() const;
 
+//////////////////////////////////////////////////////////////////////////
+// UFlowComponent overrideable events
+
+public:
+	// Called when a Root flow asset triggers a CustomOutput
+	UFUNCTION(BlueprintImplementableEvent, DisplayName = "OnTriggerRootFlowOutputEvent")
```

Comment:
Blueprint overrideable event.  The dispatcher will still call the blueprint and code versions (so that the blueprint author does not need to 'remember' to call the parent version in addition to their own).